integration_tests/pages/allocateToActivity/allocationDashboard.ts

```diff
@@ -26,7 +26,7 @@ export default class AllocationDashboard extends Page {
       })
 
   selectCandidateWithName = (name: string): void => {
-    this.getInputByLabel(name).click()
+    this.getInputByLabel(name).click({ force: true })
```

Comment:
Not sure what's going on here - the label is blocking the input from cypress. I saw something similar in Adjudications, it was to do with a cypress version, so I've updated the package for now